Integrations/integration-Zendesk.yml

```diff
@@ -511,17 +522,23 @@ script:
             ec.Ticket = newContext;
         }
 
-
         var res = {
             TicketDetails: resDetails,
             TicketComments: resComments,
             Users: usersEc,
+            Org: org,
```

Comment:
I would call it Organization and not Org.

Reply:
will fix